groclient/client.py

```diff
@@ -854,9 +886,13 @@ def get_descendant_regions(
         descendant_level=None,
         include_historical=True,
         include_details=True,
+        distance=None,
     ):
         """Look up details of all regions of the given level contained by a region.
 
+        This method is deprecated, and should be replaced by :meth:`~.get_descendant` which has
+        been updated to include the `descendant_level` and `include_historical` parameters.
```

Comment:
update get_provinces to use get_descendant

Reply:
Got it, thank you!